### PR TITLE
Changelog for 0.11.0

### PR DESCRIPTION
**Pull request by kmike**

### CODE REVIEW

File: CHANGES.rst [hunk 1]
```diff
@@ -1,6 +1,42 @@
 Changes
 =======
 
+0.11.0 (TBD)
+------------
+
+* New ref:`Articles spider template <job-posting>`, built on top of
+  Zyte API’s :http:`request:article` and :http:`request:articleNavigation`.
+
+* New ref:`Job Posting spider template <job-posting>`, built on top of
```
[the remark this answers is not in the document]
Comment (kmike) [reply CHANGES.rst:10]: hm, I wonder why isn't our docs environment failing on that?

File: CHANGES.rst [hunk 2]
```diff
@@ -1,6 +1,42 @@
...
+  Zyte API’s :http:`request:jobPosting` and :http:`request:jobPostingNavigation`.
+
+* :ref:`Search queries <search-queries>` support is added to the
+  :ref:`e-commerce spider template <e-commerce>`.
+  This allows to provide a list of search queries to the
+  spider; spider finds a search form on a website, and submits all the queries.
```
Comment (Gallaecio) [inline CHANGES.rst:16]:
```suggestion
  spider; the spider finds a search form on the target webpage, and submits all the queries.
```

File: CHANGES.rst [hunk 3]
```diff
@@ -1,6 +1,42 @@
...
+
+* ProductList extraction support is added to the
+  :ref:`e-commerce spider template <e-commerce>`. This allows spiders to
+  extract basic product information without going to product detail pages.
+
+* New features are added to the :ref:`Google Search spider template <google-search>`:
+
+    * An option to follow the result links and extract data
+      from the target pages (via ``extract`` argument)
```
Comment (Gallaecio) [inline CHANGES.rst:25]:
```suggestion
      from the target pages (via the ``extract`` argument)
```

File: CHANGES.rst [hunk 4]
```diff
@@ -1,6 +1,42 @@
...
+    * Content Languages (lr) parameter
+    * Content Countries (cr) parameter
+    * User Country (gl) parameter
+    * User Language (hl) parameter
+    * results_per_page parameter
+
+* Scrapy addon is added. This allows to greatly simplify initial
```
Comment (Gallaecio) [inline CHANGES.rst:32]:
```suggestion
* Added a Scrapy add-on. This allows to greatly simplify the initial
```

File: CHANGES.rst [hunk 5]
```diff
@@ -1,6 +1,42 @@
...
+  zyte-spider-templates configuration.
+
+* Bug fix: incorrectly extracted URLs no longer make the spiders drop
+  other requests.
```
Comment (Gallaecio) [inline CHANGES.rst:36]:
```suggestion
* Bug fix: incorrectly extracted URLs no longer make spiders drop
  other requests.
```

File: CHANGES.rst [hunk 6]
```diff
@@ -1,6 +1,42 @@
...
+
+* Cleanup CI; improved testing suite; clean up documentation.
```
Comment (Gallaecio) [inline CHANGES.rst:38]:
```suggestion
* Cleaned up the CI; improved the testing suite; cleaned up the documentation.
```

**Review by Gallaecio — APPROVED** [review]

**Pull request merged**